maintainers/maintainer-list.nix

```diff
@@ -13259,6 +13259,12 @@
     githubId = 42619;
     name = "Wei-Ming Yang";
   };
+  rickvanprim = {
+    email = "me@rickvanprim.com";
+    github = "rickvanprim";
+    githubId = 13792812;
+    name = "James Leitch";
+  };
```

Comment:
Can you please split this into a seperate commit, with subject `maintainers: add rickvanprim`?

Reply:
@matthiasbeyer done!  Just to clarify, separate commit on the same PR (which has been pushed), or create a separate PR just for adding me to the maintainers list?

Reply:
Separate commit on same PR should be fine I guess. Thanks!